css/src/components/pagination.scss

```diff
@@ -14,16 +14,16 @@ $pagination-focus-border-color: $primary-active !default;
 $pagination-active-color: $primary-active !default;
 $pagination-active-border-color: $primary-active !default;
 
-$pagination-disabled-color: $secondary !default;
-$pagination-disabled-background-color: $secondary-background !default;
-$pagination-disabled-border-color: $secondary-background !default;
+$pagination-disabled-color: $secondary-dark !default;
+$pagination-disabled-background-color: $secondary !default;
+$pagination-disabled-border-color: $secondary !default;
 
 $pagination-current-color: $text-invert !default;
 $pagination-current-background-color: $primary !default;
 $pagination-current-background-color-hover: $primary-hover !default;
 $pagination-current-border-color: $text-invert !default;
 
-$pagination-ellipsis-color: $secondary-background !default;
+$pagination-ellipsis-color: $secondary !default;
```

Comment:
it's basically invisible. Can we do something darker here please?

![image](https://github.com/user-attachments/assets/9fdda242-0863-466e-b1b8-68435e3e8092)

Reply:
Oh, sorry, I didn't realize it was like that already. I'm wondering if it's still an a11y bug

Reply:
It's ever slightly lighter than the existing gray. 

@BenDMyers  Do you foresee an a11y problem with the ellipsis color?

Reply:
Removed the ellipsis color completely.

Reply:
On its surface, this strikes me as a failure of [SC 1.4.3: Contrast (Minimum)](https://www.w3.org/WAI/WCAG22/Understanding/contrast-minimum.html), given that the ellipsis _is_ text, and it is being used to communicate that there are omitted items -- making it informational.

I think one _could_ make a case that the ellipsis is decorative/not necessary, given that one could probably guess that items are omitted by the fact that numbers are getting skipped, and that even if you can't see the ellipsis, you can see there is plenty of spacing between the left group and the right group.

Reply:
Ah, just moments too late. Thanks, Ming!